docs/source/markdown/podman-play-kube.1.md

```diff
@@ -42,6 +42,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+#### **--log-driver**=driver
+
+Logging driver for the container
+
```

Comment:
This is one that snuck in while the CI tests were disabled. I just copied the --help output; it may need expansion.

Reply:
Um. Probably "Set logging driver for all created containers"?

Reply:
Updated and pushed, thanks.